server/index.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+
+import express, {
+  Express,
+} from 'express';
+import pino from 'pino-http';
+
+import appHandler from './appHandler';
+
+const app: Express = express();
+const logger = pino();
+const port: Number = 8080;
+
+app.use(logger);
+app.use('/scripts', express.static(path.resolve(__dirname, 'scripts')));
+
+app.get('*', appHandler);
```

Comment:
This file looks much cleaner and modules have a clearer scope of responsibility! :+1: